js/modules/k6/http/request.go

```diff
@@ -310,6 +312,26 @@ func (h *HTTP) parseRequest(
 				case *HTTPCookieJar:
 					result.ActiveJar = v.jar
 				}
+			case "compression":
+				var algo = params.Get(k).ToString().String()
+				if algo != "gzip" {
```

Comment:
This would likely have to become an enum sooner or later, considering that we use it in the response handling as well: https://github.com/loadimpact/k6/blob/3506ee1aa6351f5b23b15c61d77c8dd1e37a962a/lib/netext/httpext/request.go#L246-L251

Reply:
I'm not pushing for it now, a simple `TODO` would be fine for now, but we probably should add a go-generated enumer thing (like the [response types](https://github.com/loadimpact/k6/blob/3506ee1aa6351f5b23b15c61d77c8dd1e37a962a/lib/netext/httpext/response.go#L40-L60)) for it soon, since it would simplify the validation and prevent mistakes.

Reply:
I would leave this for when/if we get a second and third option the issue in question asks for only gzip at this point, if @robingustafsson  says we could add different algorithms

Reply:
At this point we only need gzip, but in the future deflate is likely to be added.